.github/workflows/zeebe-benchmark.yml

```diff
@@ -213,6 +213,16 @@ jobs:
     name: Deploy
     needs:
       - calculate-image-tag
```

Comment:
🔧I think we can remove the dependency to the calculate tag as this is inherently?

Reply:
We need to keep it: expressions like `--set global.image.tag=${{ needs.calculate-image-tag.outputs.image-tag }}` won't work, if the needs object (`calculate-image-tag` in this example) is not explicitly listed in the `needs` list (kind of a syntax check GHA is doing there)